### PR TITLE
IO-507: Rename ByteOrderUtils class to ByteOrderParser and remove some

### PR DESCRIPTION
**Pull request by britter**
logic for parsing strings "big" and "little", after discussions on the
ML.

### CODE REVIEW

**Comment by britter** [comment]
@garydgregory what do you think?

**Comment by garydgregory** [comment]
+1

File: src/main/java/org/apache/commons/io/ByteOrderParser.java [hunk 1]
```diff
@@ -25,38 +25,26 @@
...
 
     /**
      * ByteOrderUtils is a static utility class, so prevent construction with a private constructor.
      */
-    private ByteOrderUtils() {
+    private ByteOrderParser() {
     }
 
     /**
      * Parses the String argument as a {@link ByteOrder}, ignoring case.
```
Comment (PascalSchumacher) [inline src/main/java/org/apache/commons/io/ByteOrderParser.java]: I think the pull request changes this, or I'm I wrong? Maybe we should keep the case insensitivity?

Comment (britter) [reply src/main/java/org/apache/commons/io/ByteOrderParser.java]: You're right. I've changed the documentation. Let's make it restrictive and see if users want to have a more relaxed handling.

**Comment by britter** [comment]
Fixed in master

**Pull request closed without merging**